src/quo/components/links/internal_link_card/community/view.cljs

```diff
@@ -0,0 +1,90 @@
+(ns quo.components.links.internal-link-card.community.view
+  (:require
+    [quo.components.community.community-stat.view :as community-stat]
+    [quo.components.links.internal-link-card.community.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]))
+
+(defn- description-comp
+  [description members-count active-members-count]
+  [rn/view
+   [text/text
+    {:size                :paragraph-2
+     :number-of-lines     3
+     :accessibility-label :description}
+    description]
+   [rn/view {:style style/stat-container}
+    [community-stat/view
+     {:value               members-count
+      :icon                :i/members
+      :accessibility-label :members-count
+      :style               {:margin-right 12}}]
+    (when active-members-count
+      [community-stat/view
+       {:value               active-members-count
+        :icon                :i/active-members
+        :accessibility-label :active-members-count}])]])
+
+(defn- title-comp
+  [title]
+  [text/text
+   {:size                :paragraph-1
+    :number-of-lines     1
+    :weight              :semi-bold
+    :style               style/title
+    :accessibility-label :title}
+   title])
+
+(defn- thumbnail-comp
+  [thumbnail]
+  [rn/image
+   {:style               style/thumbnail
+    :source              thumbnail
+    :accessibility-label :thumbnail}])
+
+(defn- logo-comp
+  [logo]
+  [rn/image
+   {:accessibility-label :logo
+    :source              logo
+    :style               style/logo}])
+
+(defn- stat-loading
+  [theme]
+  [rn/view {:style style/loading-stat-container}
+   [rn/view {:style (style/loading-circle theme)}]
+   [rn/view {:style (style/loading-stat theme)}]])
+
+(defn- loading-view
+  [theme]
+  [rn/view {:accessibility-label ::loading-community-link-view}
```

Comment:
I think it's best if we avoid namespaced keys in the prop `:accessibility-label` because otherwise e2e tests using them will fail when namespaces are renamed/moved. One might say the component namespaces are fairly stable, but why take the risk right? In case the namespace *path* changes in the future, the dev will need to ask QAs to update the tests or the dev will need to update the python code themselves (we will forget to do both).

Reply:
This was a typo, Apologies.